core/controllers/suggestion.py

```diff
@@ -770,10 +771,11 @@ def get(self, target_type: str, suggestion_type: str) -> None:
                 .get_suggestions_with_editable_explorations(
                     reviewable_suggestions))
         elif suggestion_type == feconf.SUGGESTION_TYPE_ADD_QUESTION:
-            suggestions, next_offset = (
-                suggestion_services
-                .get_reviewable_question_suggestions_by_offset(
-                    self.user_id, limit, offset, sort_key))
+            if limit:
```

Comment:
Instead of this, I would flip the check and throw an exception if the limit is `None`.

Reply:
Done